aptos-move/e2e-tests/src/executor.rs

```diff
@@ -642,7 +642,7 @@ impl FakeExecutor {
         )
         .unwrap();
         let remote_view = StorageAdapter::new(&self.data_store);
-        let mut session = vm.new_session(&remote_view, SessionId::void());
+        let mut session = vm.new_session(&remote_view, SessionId::void(), true);
```

Comment:
would we not want to support it here for testing?

Reply:
Instead of hard coding it to true, the `exec` and `try_exec` functions now to take `aggregator_enabled` flag as input. This lets us test when the aggregator is disabled. But there are still no end-to-end execution tests with aggregators disabled.